sdk/core/core-lro/src/http/operation.ts

```diff
@@ -54,18 +54,25 @@ function findResourceLocation(inputs: {
     case "DELETE": {
       return undefined;
     }
+    case "PATCH": {
+      return getDefault() ?? requestPath;
```

Comment:
I think a more precise modeling would be to use request path when the initial response status code is 201 but this will complicate the implementation, so let's go with this simpler version for now.